march_hardware_interface/src/march_hardware_interface.cpp

```diff
@@ -36,8 +36,9 @@ MarchHardwareInterface::MarchHardwareInterface(std::unique_ptr<march::MarchRobot
       this->joint_names_.push_back(urdf_joint.first);
     }
   }
-
   this->num_joints_ = this->joint_names_.size();
+
+  this->reset_imc_ = reset_imc;
```

Comment:
It is better to initialize this variable in the initializer list of the constructor, so the same as how `march_robot_` and `has_power_distribution_board_` are initialized. See https://en.cppreference.com/w/cpp/language/initializer_list

Reply:
nice touch